store/evm_aux/store.go

```diff
@@ -73,7 +58,21 @@ func NewEvmAuxStore(db *leveldb.DB) *EvmAuxStore {
 		}
 		dupEVMTxHashes[string(dupTxHash)] = true
 	}
+	return NewEvmAuxStore(evmAuxDB, dupEVMTxHashes), nil
+}
+
+// ChildTxRef links a Tendermint tx hash to an EVM tx hash.
+type ChildTxRef struct {
+	ParentTxHash []byte
+	ChildTxHash  []byte
+}
```

Comment:
Isn't this defined elsewhere already?

Reply:
This is a bug on github. I did not touch ChildTxRef at all.